modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/view_cp_definition_grouped_entries.jsp

```diff
@@ -190,7 +190,11 @@ renderResponse.setTitle(cpDefinition.getName(themeDisplay.getLanguageId()));
 				'<liferay-ui:message key="are-you-sure-you-want-to-delete-the-selected-entries" />'
 			)
 		) {
-			var form = window.document['<portlet:namespace />fm'];
```

Comment:
First time in my life I see this :O

Reply:
![](https://media4.giphy.com/media/8L0wy2913myHOYZlXI/giphy.gif)

Reply:
It is an old custom API. It's not a blocker, but we should use `getElementById` instead. See more details in https://github.com/liferay-frontend/liferay-portal/pull/757#discussion_r570805703